src/compas_fab/backends/pybullet/backend_features/pybullet_inverse_kinematics.py

```diff
@@ -76,57 +88,111 @@ def inverse_kinematics(self, robot, frame_WCF, start_configuration=None, group=N
         if start_configuration:
             start_configuration = self.client.set_robot_configuration(robot, start_configuration, group)
```

Comment:
this doesn't look right.  shouldn't it be set to the zero config if `start_configuration` isn't given?

Reply:
yes, i think that happens in line 92

Reply:
but the configuration is not `set` in the pybullet server.  so it uses the current pose as the seed rather than the zero config as suggested by the doc string (considering it seems that `restPoses` doesn't really have an effect).